velox/functions/sparksql/CMakeLists.txt

```diff
@@ -27,7 +27,8 @@ add_library(
   RegisterCompare.cpp
   Size.cpp
   SplitFunctions.cpp
-  String.cpp)
+  String.cpp
+  ToJson.cpp)
```

Comment:
I would suggest to use name `json.cpp` to support following json functions in https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/jsonExpressions.scala

Reply:
Renamed.